doc/source/usage.rst

```diff
@@ -32,32 +32,64 @@ formatted as a list of dictionaries mapping the variable (key) to its attributes
 .. code:: yaml
 
    - Some Variable:
-      unit: A unit
-      description: A short description
-      <other attribute>: Some text (optional)
+       unit: A unit
+       description: A short description
+       <other attribute>: Some text, value, boolean or list (optional)
 
-**Notes**
+A *unit* attribute
+^^^^^^^^^^^^^^^^^^
+
+Every variable **must have** a **unit**, which should be compatible with
+the `iam-units <https://github.com/iamconsortium/units>`_ package.
+
+The unit attribute can be empty, i.e., the variable is *dimensionless*,
+
+    .. code:: yaml
+
+       - Some Variable:
+           unit:
+
+Attributes for region processing (optional)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* By default, when applying a :class:`RegionProcessor` instance, all variables
+  are processed using the method :meth:`pyam.IamDataFrame.aggregate_region`,
+  which performs a simple summation of all subregions.
+
+* Region aggregation for a particular variable can be skipped by using the attribute
+  *skip-region-aggregation*.
+
+    .. code:: yaml
+
+       - Some Variable:
+           skip-region-aggregation: true
+
+* Any attributes which are arguments of :meth:`aggregate_region` will
+  be passed to that method, e.g., *method*, *weight*.
+
+* It is possible to rename the variable returned by the region processing
+  using a *region-aggregation* attribute, which must have a mapping of
```

Comment:
'must have a mapping' suggests that I have to provide arguments for IamDataFrame.aggregate_region. Maybe more of a conceptual question, would something like this also be possible:

```yaml
variable1:
  unit: unit1
  region-aggregation:
    - variable1_a:
```

Meaning that I just want to rename variable1 to variable1_a but otherwise aggregate normally?
If so maybe it would be good to provide and example for that as well.

Reply:
Ok, adding that feature once there is a need for it sounds good.